### PR TITLE
Add ecs fields

### PR DESCRIPTION
**Pull request by mtojek**
Changes:
* write ECS fields
* update beats repo
* convert "mm" to "px" in SVG size

Issue: https://github.com/elastic/package-registry/issues/316

### CODE REVIEW

File: dev/import-beats/fields.go [hunk 1]
```diff
@@ -72,3 +73,17 @@ func loadDatasetFields(modulePath, moduleName, datasetName string) ([]byte, erro
 	}
 	return buffer.Bytes(), nil
 }
+
+func loadEcsFields(ecsDir string) ([]fieldsTableRecord, error) {
+	ecsFieldsPath := filepath.Join(ecsDir, "generated/beats/fields.ecs.yml")
```
Comment (ruflin) [inline dev/import-beats/fields.go:78]: Good for now but at some point we should start to depend on a specific release of ECS and not master. Best same version Beats is on.

File: dev/import-beats/images_svg.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ func SvgDecodeConfig(r io.Reader) (image.Config, error) {
 		width, err = strconv.ParseFloat(svgFile.Width, 32)
 		if err != nil {
 			svgFile.Width = strings.ReplaceAll(svgFile.Width, "pt", "")
+			svgFile.Width = strings.ReplaceAll(svgFile.Width, "mm", "")
```
Comment (ruflin) [inline dev/import-beats/images_svg.go:42]: Out of curiosity, don't know much about the content of svg: Why does pt and mm have to be replaced?

File: dev/import-beats/fields.go (hunk 1 above)
Comment (mtojek) [reply dev/import-beats/fields.go:78]: At the moment, we can switch branches manually on the ECS side.

File: dev/import-beats/images_svg.go (hunk 2 above)
Comment (mtojek) [reply dev/import-beats/images_svg.go]: Icons in the Kibana and the EUI repositories have dimensions defined in various formats. Honestly I would enforce to use pixels, but most likely I don't have an influence on that :) hence, convertion.

Comment (ruflin) [reply dev/import-beats/images_svg.go]: @hbharding Any comments on the above? ^

Comment (hbharding) [reply dev/import-beats/images_svg.go]: I'm not 100% sure. EUI uses unitless values for width, height, and viewbox, so what you've done looks correct. We can use CSS on our side to make the icon bigger or smaller using pixel values.

@cchaos or @snide, I'm wondering if you have any input on this. Our package registry for ingest manager lets authors define an SVG icon for their package. Are there certain things we should we should enforce in our code like "your svg must be square", "your svg format must be minimized", etc.? Or should we just trust the author knows what they are doing and perhaps we post some guidelines/docs somewhere for authors?

Comment (snide) [reply dev/import-beats/images_svg.go]: We only use pixels. You can use an `enum` scale that pre-defines that pixel value, but it's always in pixel. As Henry alludes to, CSS can be used to provide custom widths (again in pixels) if you need to go beyond that.

For the third party logos, if you see any icons that AREN'T using pixels in their SVG itself, please make issues and we'll fix them. They came a lot of times from outside sources and people may not have stripped weird sizing formats from them.

Comment (mtojek) [reply dev/import-beats/images_svg.go]: @snide With pleasure: https://github.com/elastic/kibana/issues/62262

**Pull request merged**